pkgs/top-level/all-packages.nix

```diff
@@ -223,7 +222,8 @@ with pkgs;
         EOF
       '';
       netrcImpureEnvVars = [ "${varBase}USERNAME" "${varBase}PASSWORD" ];
-    } // passthruAttrs) // { inherit rev; };
+    } // passthruAttrs) // { inherit rev; })
+  // { meta.homepage = "${baseUrl}/"; };
```

Comment:
Could you also extract `passthruAttrs` down here?

Then I'm going to merge this.

Reply:
Did some refactoring in 5d0f7dc721997424c85624df0713c07b9cc79fc4 which makes it possible. If I didn't miss anything 5d0f7dc721997424c85624df0713c07b9cc79fc4 shouldn't change the result at all.

The refactoring was necessary to avoid the `passthruAttrs` redundancy as these attributes are required for both `fetchzip` and `fetchgit` (e.g. the `sha256` attribute).